quickwit/quickwit-proto/protos/quickwit/metastore_api.proto

```diff
@@ -186,9 +186,11 @@ message DeleteQuery {
   // If set, restrict search to documents with a `timestamp < end_timestamp``.
   optional int64 end_timestamp = 3;
   // Query text. The query language is that of tantivy.
-  string query = 4;
-  // Search fields.
-  repeated string search_fields = 5;
+  // Query AST serialized in JSON
+  string query_ast = 6;
+
+  // Reserved field due to deprecation
```

Comment:
Nitpick: given that all our breaking changes, I'm wondering if we really need to keep the deprecated reserved field.

Reply:
it is absolutely useless at the moment but considering how immature we are about backward compatibility, I think a repetition of the good practise is a good idea.